docker/grafana/dashboards/RabbitMQ-Overview.json

```diff
@@ -2677,7 +2677,7 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "sum(rate(rabbitmq_channel_get{cluster=\"$cluster\"}[1m])) by(node)",
+          "expr": "sum(rate(rabbitmq_channel_get_total{cluster=\"$cluster\"}[1m])) by(node)",
```

Comment:
You should use a consistent range, and given you recommend scrape intervals of 30s 2m is the lowest safe value per https://www.robustperception.io/what-range-should-i-use-with-rate

Reply:
We actually scrape every 5s, which is a RabbitMQ Management default.

This is why the interval range differs: https://github.com/rabbitmq/rabbitmq-prometheus/commit/3b860bca949fa8ff5023c891d9f9fb122d8de500

Reply:
Your documentation recommends 30s in production though. 5s would be considered quite aggressive in general Prometheus terms.

Reply:
OK, agreed.

Reply:
15s feels more appropriate: https://github.com/rabbitmq/rabbitmq-website/commit/a929f22233842fe8b2046d1bc870a6b2ef7a23dd

Ideally, there would one value for dev, where quick feedback is preferable, and a different value for production. Would you recommend templating the interval used in `rate()`?

Reply:
$__interval in Grafana is probably the best thing here.

Reply:
Yes, `$__interval` with a min value of `60s`